### DISCUSSION TITLE
<title> Github pages multiple html

### DISCUSSION THREAD

**Discussion by WilsonMedina** [community/community#46518] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I'm uploading my personal page to github pages, the work is made up of an index.html and inside 2 other folders there are 5 other html documents with different names, the problem is that the page loads only the index.html and when I browse the menu of the same that takes me to the other files gives me a github error but running it from local everything works fine, so my question is if github pages allow multiple html files?

**Comment by SilasRodrigues19** [comment]
Hey @WilsonMedina 

You need to provide folder directory and file name with ` .html` extension as below:

https://wilsonmedina.github.io/MyPersonalSite/spanish/abuot-me-es.html

**Reply by SilasRodrigues19** [reply]
One detail you have to pay attention to is that the deploy root points to `https://wilsonmedina.github.io`, so you need to provide the path `/MyPersonalSite` for the site to understand that the root must be `https://wilsonmedina.github.io/MyPersonalSite`

**Reply by Anthonia-alexis** [reply]
please, I still don't understand. my page keeps loading only my readme.md file. how do I make it load my html files?

**Reply by SilasRodrigues19** [reply]
@Anthonia-alexis Hi

Keep your main html file named index.html, if the file name is different from `index.html` you need to enter the name followed by a slash after your domain. The server always looks by default for a file named `index`: `index.html`, `index.php`, etc.

Example:
anthonia-alexis.github.io/form/ -> loads the `index.html` (by default)
anthonia-alexis.github.io/form`/terms-and-conditions.html`-> loads the `terms-and-conditions.html`
anthonia-alexis.github.io/form`/myHtmlFile.html` -> loads the `myHtmlFile.html`

**Reply by Anthonia-alexis** [reply]
Alright. I'll set it up as you explained. I'll drop a msg if things turn out great. 

Thank you.

**Comment by WilsonMedina** [comment]
Thanks

**Reply by SilasRodrigues19** [reply]
It was nothing, please mark as answered if it solves your question.
